en/flying/led_meanings.md

```diff
@@ -0,0 +1,38 @@
+# LED meanings
+
+[Pixhawk flight controllers](../flight_controller/pixhawk_series.md) have an RGB LED to indicate the current status of the FMU.
```

Comment:
* [Pixhawk flight controllers] -> [Pixhawk-series flight controllers]
* "current status of the FMU." -> "current status of the vehicle."